Frontend/android-native/app/src/driver/nammaYatriPartner/res/layout/splash.xml

```diff
@@ -2,44 +2,47 @@
 <androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:background="@drawable/ny_ic_splash_bg"
```

Comment:
if this is removed, can we delete this asset as well? is it being used somewhere else?

Reply:
It is used in customer splash screen.